Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandGroupMemberKick.java

```diff
@@ -1,88 +1,95 @@
-package com.universeprojects.miniup.server.commands;
```

Comment:
This entire file is shown as different. It seems to happen sometimes but I don't know why. I'll need to see a proper diff before I can approve. Try reverting and making your changes again?